documentation/how-to-use.md

```diff
@@ -0,0 +1,78 @@
+In this document, we will introduce how to use JS SDK for very new beginner to typescript SDK.
```

Comment:
we will give a brief introduction on how to use the JavaScript SDK for new users